packages/replay-next/components/inspector/protocolValueToText.ts

```diff
@@ -1,89 +1,196 @@
 import {
+  ObjectId,
   PauseId,
   NamedValue as ProtocolNamedValue,
   Value as ProtocolValue,
 } from "@replayio/protocol";
 
-import { ReplayClientInterface } from "shared/client/types";
-
 import { dateProtocolObjectToString } from "replay-next/components/inspector/values/DateRenderer";
 import { errorProtocolObjectToString } from "replay-next/components/inspector/values/ErrorRenderer";
+import { functionProtocolObjectToString } from "replay-next/components/inspector/values/FunctionRenderer";
 import { regExpProtocolObjectToString } from "replay-next/components/inspector/values/RegExpRenderer";
 import { getObjectWithPreviewHelper } from "replay-next/src/suspense/ObjectPreviews";
 import { protocolValueToClientValue } from "replay-next/src/utils/protocol";
-import { functionProtocolObjectToString } from "replay-next/components/inspector/values/FunctionRenderer";
+import { ReplayClientInterface } from "shared/client/types";
+
+const MAX_DEPTH_TO_COPY = 5;
```

Comment:
:speech_balloon: probably good enough for now. I guess if they want more depth they can right-click on a more deeply-nested field and grab that?

(Could also consider adding a secondary context option to copy _everything_.)

Reply:
> (Could also consider adding a secondary context option to copy everything.)

I think I'd rather avoid that. The UX feels awkward.

>  I guess if they want more depth they can right-click on a more deeply-nested field and grab that?

Yes. You can expand an object and copy any of its properties.